File: Activity/Activity.Core/Entities/taskStatus.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Activity.Core.Entities
{
    public class taskStatus : baseEntity
    {
        public int CompanyId { get; set; }
        public int ProjectId { get; set; }
        public string TaskStatusName { get; set; }
    }

    public class taskStatusRequest : taskStatus
    {
        public int TaskStatusId { get; set; }
        public int CreationUserId { get; set; }
        public bool Active { get; set; }
    }
    public class taskStatusResponse : taskStatus
    {
        public int TaskStatusId { get; set; }
        public DateTime CreationDate { get; set; }
    }


}

File: Web/src/app/interfaces/taskStatus.ts
export interface taskStatus {
    companyId: number;
    projectId: number;
    taskStatusId: number;
    taskStatusName: string;
    description: string | null;
    creationDate: Date;
}


export interface taskStatusRequest {
    companyId: number;
    projectId: number;
    taskStatusId: number;
    taskStatusName: string;
    CreationUserId: number;
    Active: boolean;
}